Sources/Commands/SwiftTool.swift

```diff
@@ -55,8 +55,10 @@ struct TargetNotFoundDiagnostic: DiagnosticData {
     let targetName: String
 }
 
-private class ToolWorkspaceDelegate: WorkspaceDelegate {
-
+internal class ToolWorkspaceDelegate: WorkspaceDelegate {
+    
+    var shouldOutputToStdErr: Bool = false
```

Comment:
nit: doc comment pls

Reply:
I'll pick up these comments, thanks for the review :)